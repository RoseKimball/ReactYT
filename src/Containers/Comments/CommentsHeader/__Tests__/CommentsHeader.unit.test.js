import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import CommentsHeader from '../CommentsHeader';

describe('CommentsHeader', () => {
    test('renders CommentsHeader', () => {
        const wrapper = shallow (
            <CommentsHeader />
        )
        expect(wrapper).toMatchSnapshot();
    })
    test('renders CommentsHeader with amountComments prop', () => {
        const wrapper = shallow (
            <CommentsHeader amountComments={1}/>
        )
        expect(wrapper).toMatchSnapshot();
    }) 
})