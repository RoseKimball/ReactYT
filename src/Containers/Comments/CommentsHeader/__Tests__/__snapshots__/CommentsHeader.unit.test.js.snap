// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader renders CommentsHeader 1`] = `ShallowWrapper {}`;

exports[`CommentsHeader renders CommentsHeader with amountComments prop 1`] = `ShallowWrapper {}`;
